app/src/components/sections/SingleTeamView/BankSection.vue

```diff
@@ -386,6 +385,7 @@ const { isLoading: isConfirmingPushTip, isSuccess: isConfirmedPushTip } =
 watch(isConfirmingPushTip, (isConfirming, wasConfirming) => {
   if (wasConfirming && !isConfirming && isConfirmedPushTip.value) {
     addSuccessToast('Tips pushed successfully')
+    fetchBalance()
```

Comment:
I think you don't need this, because normally with wagmi, the balance will update automatically right ?